GiniVisionExample/GiniVisionExample/ViewControllers/HelpViewController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  HelpViewController.swift
+//  GiniVisionExample
+//
+//  Created by Enrique del Pozo Gómez on 11/21/17.
+//  Copyright © 2017 Gini. All rights reserved.
+//
+
+import UIKit
+import GiniVision
+
+typealias HelpLink = (title: String, url: URL?)
+typealias HelpVersion = (title: String, version: String)
+
+protocol HelpViewControllerDelegate: class {
+    func help(viewController: HelpViewController, didSelectLink link: HelpLink)
+    func help(viewController: HelpViewController, didTapClose: ())
+}
+
+final class HelpViewController: UIViewController {
+    weak var delegate: HelpViewControllerDelegate?
+    let linkCellReuseIdentifier = "linkCellReuseIdentifier"
+    let versionCellReuseIdentifier = "versionCellReuseIdentifier"
+    
+    let versions: [HelpVersion] = [("GVL Version", AppHelper.gvlVersion ?? ""),
```

Comment:
Do you think it's possible that the app cannot get it's own version? I'd hope that you can make those methods return non-optionals.

Reply:
The only way is moving the unwrap to AppHelper class, because you need to cast it and I don't want to force cast tbh.